patches/common/chromium/global_menu_bar_registrar_x11.patch

```diff
@@ -0,0 +1,41 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Milan Burda <miburda@microsoft.com>
+Date: Fri, 26 Oct 2018 20:10:06 +0200
+Subject: global_menu_bar_registrar_x11.patch
+
+Use atom::GlobalMenuBarX11::GetPathForWindow
+
+diff --git a/chrome/browser/ui/views/frame/global_menu_bar_registrar_x11.cc b/chrome/browser/ui/views/frame/global_menu_bar_registrar_x11.cc
+index b112b0f2f81c823856dbce01e75f1bb4cbad03bf..cead675a74d33ac990b1a77fe5f9c2c6416ac249 100644
+--- a/chrome/browser/ui/views/frame/global_menu_bar_registrar_x11.cc
++++ b/chrome/browser/ui/views/frame/global_menu_bar_registrar_x11.cc
+@@ -4,10 +4,10 @@
+ 
+ #include "chrome/browser/ui/views/frame/global_menu_bar_registrar_x11.h"
+ 
++#include "atom/browser/ui/views/global_menu_bar_x11.h"
+ #include "base/bind.h"
+ #include "base/debug/leak_annotations.h"
+ #include "base/logging.h"
+-#include "chrome/browser/ui/views/frame/global_menu_bar_x11.h"
+ #include "content/public/browser/browser_thread.h"
+ 
+ using content::BrowserThread;
+@@ -70,7 +70,7 @@ GlobalMenuBarRegistrarX11::~GlobalMenuBarRegistrarX11() {
+ 
+ void GlobalMenuBarRegistrarX11::RegisterXID(unsigned long xid) {
+   DCHECK(registrar_proxy_);
+-  std::string path = GlobalMenuBarX11::GetPathForWindow(xid);
++  std::string path = atom::GlobalMenuBarX11::GetPathForWindow(xid);
```

Comment:
We haven't made patches with dependency from `//electron` inside the `//chrome`, not sure if this allowed. Another way to solve this would be to implement it via `g_browser_process`.  /cc @nornagon

Reply:
Agreed, let's find another way to do this.

Reply:
keeping `global_menu_bar_registrar_x11.h` in `chromium_src` for now